palak/src/components/Button.jsx

```diff
@@ -0,0 +1,35 @@
+// import { Children } from "react";
+// import withLoading from "./withLoading";
+const Button = ({ onClick, className, children }) => {
+  return (
+    <button
+      className={`text-sm font-medium h-11 rounded-md w-full focus-visible:outline-none focus-visible:ring-2  max-w-sm mt-10 text-white bg-blue-800 hover:bg-blue-700 ${className}`}
```

Comment:
If a button with similar properties is being used at multiple places, then try to make it a reusable component.